Sources/Danger/GitLabDSL.swift

```diff
@@ -178,7 +178,7 @@ extension GitLab {
         public let downvotes: Int
         public let firstDeployedToProductionAt: Date?
         public let forceRemoveSourceBranch: Bool
-        public let id: Int // swiftlint:disable:this identifier_name
+        public let id: Int 
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/GitLabDSL.swift;
  Line: 181;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)